src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1573,15 +1570,15 @@ bool Graph::InsertNode(EdgePtr edge, NodePtr node, bool initNode) {
                        " and ",
                        edge->getChild()->getName(),
                        ".");
-    edge->drop();
+    edge->getParent()->removeChildEdge(edge);
```

Comment:
Don't we want to use `Graph::RemoveEdge` here?

Reply:
I have tried to use RemoveEdge but it leads to an error which is not easy to debug.
I will try once more after all the fixes are done.